l10n_ch_pain_base/models/account_payment_order.py

```diff
@@ -89,7 +89,7 @@ def generate_party_acc_number(self, parent_node, party_type, order,
             if not partner_bank.ccp:
                 raise UserError(_(
                     "The field 'CCP/CP-Konto' is not set on the bank "
-                    "account '%s'.") % partner_bank.name)
+                    "account '%s'.") % partner_bank.bank_id.name)
```

Comment:
I think it should be `partner_bank.acc_number` instead. bank_id could even not be set.
The message is there to help identify which bank account is messing up with the pain generation.